cli/eventstore/eventstore.js

```diff
@@ -0,0 +1,56 @@
+var path = require('path')
+
+var { TransmuteFramework } = require('../../build/TransmuteFramework')
+
+let { web3, EventStoreContract, EventStoreFactoryContract } = TransmuteFramework.init()
+
+// These should be updated to be permissions and default event oriented
+var { readModel, reducer } = require('./legacy/reducer')
+
+var _ = require('lodash')
+
+// transmute eventstore create --from 0x23b...
+// - create a new event store
+const createEventStore = async (fromAddress) => {
+    const esf = TransmuteFramework.EventStore.EventStoreFactory
+    const factory = await EventStoreFactoryContract.deployed()
+    const { tx, events } = await esf.createEventStore(factory, fromAddress)
+    const newEsAddress = events[0].AddressValue
+    return newEsAddress
+}
+
+const getCachedReadModel = async (contractAddress, eventStore, fromAddress, readModel, reducer) =>{
+  readModel.readModelStoreKey = `${readModel.readModelType}:${contractAddress}`
+  readModel.contractAddress = contractAddress
+  let maybeSyncReadModel = TransmuteFramework.EventStore.ReadModel.maybeSyncReadModel
```

Comment:
you prefer {maybeSyncReadModel } = TransmuteFramework.EventStore.ReadModel ?

I think that is only necessary when the line length gets too long.